### ISSUE TITLE
Galactic Materials still has an infinite money glitch

### ISSUE THREAD

**Issue by StrangeWeirdKitten** [tgstation/tgstation#79978]
![image](https://github.com/tgstation/tgstation/assets/95130227/3875620b-e10a-43ca-b498-955cb8f63068)
Our stockmarket code between both skyrat and us is untouched and fully up to date with the latest fixes. Our latest upstream merge was the 20th
## Reproduction:
![image](https://github.com/tgstation/tgstation/assets/95130227/db4eafa6-8180-4906-963b-0b879285c8d8)

Buy diamonds low, sell them at high. This screenshot basically shows the insane amounts of diamonds you can buy and sell off for infinite stonks

![image](https://github.com/tgstation/tgstation/assets/95130227/6008208d-c8bb-4203-8acd-6e577fd19051)

**Comment by deathrobotpunch** [comment]
isn’t that how the stock market works IRL

**Comment by the-orange-cow** [comment]
Is this what they meant by diamond hands?

**Comment by SyncIt21** [comment]
We have a number of options

1. Make sure the materials price and quantity don't cross a certain threshold during both markets fluctuations and purchases.

2. Decrease the cost of a material by a large margin after selling a single sheet/stack so that subsequent selling doesn't yield large profits 

3. Count the total number of sheets from all the crates in the shuttle rather than selling them one at a time and decide the final price based on that bulk amount 

@ArcaneMusic which would you prefer?

**Comment by deathrobotpunch** [comment]
number 2 would mimic how a stock market works IRL

**Comment by deathrobotpunch** [comment]
but it would be best if the price increased steadily the more you buy

**Comment by Admiral-Gigaton** [comment]
me when the coders make the stonks market too realistic

**Comment by ArcaneMusic** [comment]
> We have a number of options
> 
> 1. Make sure the materials price and quantity don't cross a certain threshold during both markets fluctuations and purchases.
> 2. Decrease the cost of a material by a large margin after selling a single sheet/stack so that subsequent selling don't yield large profits
> 3. Count the total number of sheets from all the crates in the shuttle rather than selling them one at a time and decide the final price based on that bulk amount
> 
> @ArcaneMusic which would you prefer?

Yeah, #2 is both the way to go and also was what I attemped to implement in the first place. #1 we also have to a certain extent as well. But lets crank up the numbers on #2 and see where that leaves us.

**Comment by adamkad1** [comment]
Hnnnng, Fun hating.
